node-module-manager.js

```diff
@@ -85,6 +86,10 @@ me.loadModules = function(cb) {
 me.initModules = function(cb) {
 	// Todo, don't ever init if they're already initted
 	_.each(_.values(modules), function(mod) {
```

Comment:
Can you modify this loop to capture the key (module name) of each module? That would eliminate the `moduleName` function below.